File: src/test/java/com/survivalMode/SurvivalModePluginTest.java
package com.survivalMode;

import net.runelite.client.RuneLite;
import net.runelite.client.externalplugins.ExternalPluginManager;

public class SurvivalModePluginTest
{
	public static void main(String[] args) throws Exception
	{
		ExternalPluginManager.loadBuiltin(SurvivalModePlugin.class);
		RuneLite.main(args);
	}
}

File: src/test/java/com/survivalMode/SurvivalModePluginTest.kt
package com.survivalMode

import net.runelite.client.RuneLite
import net.runelite.client.externalplugins.ExternalPluginManager

object SurvivalModePluginTest {
    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        ExternalPluginManager.loadBuiltin(SurvivalModePlugin::class.java)
        RuneLite.main(args)
    }
}